### PR TITLE
Wizard recipe: libjwt-v1.15.3

### PR DESCRIPTION
**Pull request by philip-ittmann**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: libjwt
* Version: v1.15.3

@staticfloat please review and merge.

### CODE REVIEW

File: L/libjwt/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libjwt", :libjwt),
+    ExecutableProduct("jwtgen", :jwtgen),
+    ExecutableProduct("jwtauth", :jwtauth)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenSSL_jll", uuid="458c3c95-2e84-50aa-8efc-19380b2a3a95"))
```
Comment (giordano) [inline L/libjwt/build_tarballs.jl:41]:
```suggestion
    Dependency(PackageSpec(name="OpenSSL_jll", uuid="458c3c95-2e84-50aa-8efc-19380b2a3a95"); compat="3.0.8")
```

File: L/libjwt/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "linux"; libc = "glibc")
+]
```
Comment (giordano) [inline L/libjwt/build_tarballs.jl:29]: Any reason for not supporting more platforms?

File: L/libjwt/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+sources = [
+    GitSource("https://github.com/benmcollins/libjwt.git", "e8af37ce2e7de2d3bbadaaf232dc6f6b0fd97f03")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libjwt
+autoreconf -i
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make
```
Comment (giordano) [inline L/libjwt/build_tarballs.jl:18]:
```suggestion
make -j${nproc}
```

**Review by giordano — COMMENTED** [review]
Thanks for your contribution!

File: L/libjwt/build_tarballs.jl (hunk 2 above)
Comment (philip-ittmann) [reply L/libjwt/build_tarballs.jl:29]: This was what I needed for my purposes, I will increase coverage now!

**Pull request merged**